dist/stat_data.py

```diff
@@ -397,6 +397,27 @@ def __init__(self, name, desc, flags=''):
     HistoryStat('hs_gc_pages_evict', 'history pages added for eviction during garbage collection'),
     HistoryStat('hs_gc_pages_removed', 'history pages removed for garbage collection'),
     HistoryStat('hs_gc_pages_visited', 'history pages visited for garbage collection'),
+    HistoryStat('hs_write_addr_prepared', 'page addresses written that included at least one prepare state'),
+    HistoryStat('hs_write_addr_newest_start_durable_ts', 'page addresses written that include a newest start durable timestamp '),
```

Comment:
I agree pretty strongly with @hbokhari, "hs" is a history store prefix, and these aren't history store statistics.

Reconciliation, maybe?